docs/samples/modelmesh_sdk_sample.ipynb

```diff
@@ -0,0 +1,562 @@
+    "\n",
+    "- This sample assumes ModelMesh Serving was deployed using the [quickstart guide](https://github.com/kserve/modelmesh-serving/blob/main/docs/quickstart.md)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kubernetes import client\n",
+    "from kserve import constants\n",
+    "from kserve import V1beta1InferenceService\n",
+    "from kserve import V1beta1InferenceServiceSpec\n",
+    "from kserve import V1beta1PredictorSpec\n",
+    "from kserve import V1beta1SKLearnSpec\n",
+    "from kserve import V1beta1TFServingSpec\n",
+    "from kserve import V1beta1StorageSpec\n",
+    "from kserve import KServeClient"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The variables being set are `namespace`, `name`, `protocol_version`\n",
+    "- The `namespace` definition is where the InferenceService will be deployed to\n",
+    "- `name` will be the name of the InferenceService\n",
+    "- For ModelMesh, the `v2` protocol must be used since it doesn't support the default `v1` protocol"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "namespace = 'modelmesh-serving'\n",
+    "name='mnist-sample'\n",
+    "protocol_version='v2'"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define InferenceService specifying ModelMesh as the deploymentMode"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "isvc = V1beta1InferenceService(\n",
+    "    api_version=constants.KSERVE_V1BETA1,\n",
+    "    kind=constants.KSERVE_KIND,\n",
+    "    metadata=client.V1ObjectMeta(\n",
+    "        name=name, \n",
+    "        namespace=namespace,\n",
+    "        annotations={\n",
+    "            'serving.kserve.io/deploymentMode': 'ModelMesh'\n",
+    "        }\n",
+    "    ),\n",
+    "    spec=V1beta1InferenceServiceSpec(\n",
+    "        predictor=V1beta1PredictorSpec(\n",
+    "            sklearn=V1beta1SKLearnSpec(\n",
+    "                protocol_version=protocol_version,\n",
+    "                storage=V1beta1StorageSpec(\n",
+    "                    key='localMinIO',\n",
+    "                    path='sklearn/mnist-svm.joblib'\n",
+    "                )\n",
+    "            )\n",
+    "        )\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create InferenceService"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'apiVersion': 'serving.kserve.io/v1beta1',\n",
```

Comment:
I don't like these very long JSON outputs. Could we update the code blocks (here and below) to capture the output objects/maps/JSON data and use the Python map or Python JSON libraries to only print relevant text or parts of the output we care about, like the deployment status, the name/id of created objects, etc.?

Reply:
@ckadner I'm not sure how to do this that you're suggesting here to compress the json outputs. Can you share an example?

Reply:
like what you do here:

```Python
print("status: ", updated["status"]["modelStatus"]["states"]["activeModelState"])
```

```
status:  Loaded
```

Or even just:

```Python
print(updated["status"]["modelStatus"]["states"]["activeModelState"])
```

```
Loaded
```

And only the parts that changed, are new ... no need to repeatedly print the same model ID / name if that already was outputted in a cell above